README.md

```diff
@@ -47,6 +47,7 @@ NOTE: This has been tested with npm version 6.14.11 and node version 14.15.5
 
 ## Settings
 - `systemverilog.includeIndexing`: _Array_, Globs defining files to be indexed
+- `systemverilog.libraryIndexing`: _Array_, Globs defining library files to be added to verilator linting. It's useful when module name is not equal to filename.
```

Comment:
Might be a good idea to add a quick note in Troubleshooting section asking users to make sure that this setting is correctly set for their repo since the default value is not-empty.

Reply:
I add a new item in Troubleshooting section to describe the problem of "modules cannot be resolve".